xem

```diff
@@ -20,6 +20,7 @@ xem is a Scala program wrapped in a shell script. It takes the source files dire
         --mask-elements             [a comma-separated list of XML element tags, whose content and sub elements should be masked with CDATA]
         --target-files-directory    [ABSOLUTE PATH TO TARGET FILES DIRECTORY]
         --content-only              utilise only the content of the sub elements (i.e. without further XML elements tags inside)
+        --exclude-empty-elements    exclude empty XML element tags, which should be masked with CDATA
```

Comment:
Well the "enhanced" XML is still valid. With the _"--exclude-empty-elements"_ option you decide if you want to have the empty XML element from the origin XML in the enhanced XML as well or not.